oh_staff_ui/tests.py

```diff
@@ -734,3 +736,24 @@ def test_post_to_item_form(self):
         form = ProjectItemForm(request.POST, parent_item=self.series_item)
         # Check that form validation succeeded.
         self.assertEquals({}, form.errors)
+
+    def test_valid_series_item_mods(self):
```

Comment:
These new tests should go into a new test case, since they don't really correspond to `ProjectItemFormTestCase`.  Make a new test case and copy `setUpTestData` , then move these tests to that case.

Reply:
`ModsTestCase` class created and cases moved